package/MDAnalysis/analysis/msd.py

```diff
@@ -299,10 +313,14 @@ class EinsteinMSD(AnalysisBase):
         Number of particles MSD was calculated over.
 
 
+    .. versionchanged:: 2.4.0
```

Comment:
Change docs here and above to be show optional

Reply:
Do you mean like this?

Reply:
Yep perfect.